packages/portal-proto/src/features/cases/CasesView/CasesCohortButton.tsx

```diff
@@ -134,7 +134,8 @@ export const CasesCohortButtonFromValues: React.FC<
 > = ({ pickedCases }: CasesCohortButtonFromValuesProps) => {
   const [createSet, response] = useCreateCaseSetFromValuesMutation();
   const onCreateSet = useCallback(
-    () => createSet({ values: pickedCases }),
+    () =>
+      createSet({ values: pickedCases, intent: "user", set_type: "frozen" }),
     [createSet, pickedCases],
```

Comment:
<img width="1422" alt="Screenshot 2024-04-05 at 2 44 08 PM" src="https://github.com/NCI-GDC/gdc-frontend-framework/assets/4624053/89036502-a975-4ea8-bad6-777e5c39648f">

Reply:
"portal" due to grey area. https://cdis.slack.com/archives/C01PGJ5KKHD/p1711987742935879?thread_ts=1711729341.125529&cid=C01PGJ5KKHD

Reply:
i don't see intent and set_type show up in the sets created when I use the 2nd and 3rd options: "Existing Cohort With/Without..."

Reply:
There is a create set query in `packages/portal-proto/src/features/cases/CasesView/SelectCohortsModal.tsx`. I Iadded the intent and type